packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,416 @@
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+The `selectionMode` prop can be used to switch from the default selection of a single menu item to multiple menu items or none.
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(['Kangaroo', 'Snake']);
+
+  let onAction = (value) => {
+    if (state.includes(value)) {
+      setState(state.filter((item) => item !== value));
+    } else {
+      let curState = state;
+      curState.push(value);
+      setState(curState);
+    }
+  };
+
+  return (
+    <div>
+      <MenuTrigger>
+        <ActionButton>
+            Menu Button Multiple
+        </ActionButton>
+        <Menu
+          items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+          itemKey="name"
+          onAction={onAction}
+          defaultSelectedKeys={state}
+          selectionMode="multiple">
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+      <MenuTrigger>
+        <ActionButton>
+            Menu Button None
+        </ActionButton>
+        <Menu
+          items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+          itemKey="name"
+          selectionMode="none">
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+    </div>
+  );
+}
+```
+
+### Sections
+
+Menu's may have Sections with titles and children items.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Section 1', children: [{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}, {name: 'Section 2', children: [{name: 'Koala'}]}]} itemKey="name">
+    {item => (
+      <Section items={item.children} title={item.name}>
+        {item => <Item>{item.name}</Item>}
+      </Section>
+    )}
+  </Menu>
+</MenuTrigger>
+```
+
+### Sematic Elements
+
+A Menu item's content may be any renderable node, not just strings.
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Content-types)
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu
+    itemKey="name"
+    items={[
+      {name: 'Copy', icon: 'Copy', shortcut: '⌘C'},
+      {name: 'Cut', icon: 'Cut', shortcut: '⌘X'},
+      {name: 'Paste', icon: 'Paste', shortcut: '⌘V'}
+    ]}>
+    {item => {
+      let iconMap = {
+        Copy,
+        Cut,
+        Paste
+      };
+      let Icon = iconMap[item.icon];
+      return (
+        <Item childItems={item.children} textValue={item.name}>
+          <Icon size="S" />
+          <Text>{item.name}</Text>
+          <Keyboard>{item.shortcut}</Keyboard>
+        </Item>
+      );
+    }}
+  </Menu>
+</MenuTrigger>
+```
+
+### Accessibility
+
+The MenuTrigger uses the prop `aria-controls` to point at the menu, which uses
+the prop `aria-labelledby` to reference the trigger. Other aria props of the
+MenuTrigger are `aria-haspopup` and `aria-expanded`.
```

Comment:
IMO, I feel like we can drop this bit since we handle this internally and I don't think we allow users to set these. The aria-label bit can stay though